### PR TITLE
AB#21 Ensure that monitoring of the site is implemented

### PR DESCRIPTION
**Pull request by Varski**
This is highly important

### CODE REVIEW

**Review by El-SaMa — CHANGES_REQUESTED** [review]
Some love is required

**Review by tiibuturner — CHANGES_REQUESTED** [review]
I think you should add . (dot) at the end of the sentence.

File: This_is_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Good text very nice
```
Comment (tiibuturner) [inline This_is_file.txt:1]:
```suggestion
Good text very nice.
```

**Review by Varski — COMMENTED** [review]
COME SEE THIS AGAIN !!!!!

**Review by tiibuturner — APPROVED** [review]

**Review by El-SaMa — APPROVED** [review]

**Pull request merged**